pkg/venafi/tpp/connector.go

```diff
@@ -1202,6 +1202,11 @@ func (c *Connector) RetrieveCertificate(req *certificate.Request) (certificates
 	includeChain := req.ChainOption != certificate.ChainOptionIgnore
 	rootFirstOrder := includeChain && req.ChainOption == certificate.ChainOptionRootFirst
 
+	// if Request doesn't contain a Format, use defaults
```

Comment:
I think this condition should go back to where it originally was. Below line 1232 `if req.CsrOrigin == certificate.ServiceGeneratedCSR || req.FetchPrivateKey {`

Otherwise we are always setting certificate format to `Base64 PKCS8` when the key is `RSA`. 
The default value is already provided by FormatNotSet enum var, so no conditional is necessary before setting the fomrat.

 `PKCS8` must only happen when the key type is `RSA` and the csrOrigin is `Service`, or the user chooses to use it.